src/notelistview.cpp

```diff
@@ -752,8 +751,8 @@ void NoteListView::onCustomContextMenu(QPoint point)
             const auto tagIds = m_tagPool->tagIds();
             for (const auto &id : tagIds) {
                 bool all = true;
-                for (const auto &index : qAsConst(indexList)) {
-                    auto tags = index.data(NoteListModel::NoteTagsList).value<QSet<int>>();
+                for (const auto &idx : qAsConst(indexList)) {
```

Comment:
Rather than creating variations of generic names, we should prefer specific names. I think in this case `selectedIndex` would be suitable (and in some other places, as far as I could see).

Renaming `indexList` to `selectedIndexes` would be good, but not necessarily in this patch.

Reply:
> I think in this case `selectedIndex` would be suitable (and in some other places, as far as I could see).

Agreed. 

> Renaming `indexList` to `selectedIndexes` would be good, but not necessarily in this patch.

Yeah, let's leave that one for our future selves... ^_^